nautobot_firewall_models/migrations/0002_custom_status.py

```diff
@@ -2,69 +2,76 @@
-                ct = ContentType.objects.get_for_model(model)
-                status.content_types.remove(ct)
+    Status = apps.get_model("extras.Status")
+    firewall_models_content_types = ContentType.objects.filter(app_label="nautobot_firewall_models")
+    for status in Status.objects.filter(content_types__in=firewall_models_content_types).distinct().iterator():
+        status.content_types.remove(*firewall_models_content_types)
 
 
 def create_default_objects(apps, schema_editor):
     """Initial subset of commonly used objects."""
-    defaults = os.path.join(os.path.dirname(__file__), "services.yml")
-    with open(defaults, "r") as f:
-        services = yaml.safe_load(f)
-    status = apps.get_model("extras.Status").objects.get(name="Active")
+    default_services_file = os.path.join(os.path.dirname(__file__), "services.yml")
+    Status = apps.get_model("extras.Status")
+    ServiceObject = apps.get_model("nautobot_firewall_models.ServiceObject")
+    default_status = Status.objects.get(name=get_default_status_name())
 
-    for i in services:
-        apps.get_model("nautobot_firewall_models.ServiceObject").objects.create(status=status, **i)
+    with open(default_services_file, "r") as f:
+        default_services = yaml.safe_load(f)
+
+    for service in default_services:
+        ServiceObject.objects.create(status=default_status, **service)
 
 
 def reverse_create_default_objects(apps, schema_editor):
-    """Removes commonly used objects."""
-    defaults = os.path.join(os.path.dirname(__file__), "services.yml")
-    with open(defaults, "r") as f:
-        services = yaml.safe_load(f)
-    status = apps.get_model("extras.Status").objects.get(name="Active")
+    """
+    Removes commonly used objects.
+
+    Currently skipped due to Django bug https://code.djangoproject.com/ticket/33586
+    """
+    default_services_file = os.path.join(os.path.dirname(__file__), "services.yml")
+    ServiceObject = apps.get_model("nautobot_firewall_models.ServiceObject")
+
+    with open(default_services_file, "r") as f:
+        default_services = yaml.safe_load(f)
 
-    for i in services:
-        try:
-            service = apps.get_model("nautobot_firewall_models.ServiceObject").objects.get(status=status, **i)
-            service.delete()
-        except ObjectDoesNotExist:
-            continue
+    for service in default_services:
+        ServiceObject.objects.filter(**service).delete()
 
 
 class Migration(migrations.Migration):
     dependencies = [
+        ("contenttypes", "0002_remove_content_type_name"),
         ("extras", "0033_add__optimized_indexing"),
         ("nautobot_firewall_models", "0001_initial"),
     ]
 
     operations = [
         migrations.RunPython(code=create_status, reverse_code=reverse_create_status),
-        migrations.RunPython(code=create_default_objects, reverse_code=reverse_create_default_objects),
+        migrations.RunPython(code=create_default_objects, reverse_code=migrations.RunPython.noop),
```

Comment:
These objects would get deleted when reversing to `Zero` anyways, I believe this is acceptable and the `reverse_create_default_objects` function can be removed.